dmaws/build.py

```diff
@@ -21,6 +23,25 @@ def run_git_cmd(args, cwd, stdout=None):
     )
 
 
+def clone_or_update(app_name):
+    app_git_url = get_application_git_url(app_name)
+    repository_path = os.path.join(REPOS_PATH,
+                                   'digitalmarketplace-{}'.format(app_name))
+    if not os.path.exists(REPOS_PATH):
+        os.mkdir(REPOS_PATH)
+    if not os.path.exists(repository_path):
+        run_git_cmd(['clone', app_git_url], REPOS_PATH)
+    else:
+        run_git_cmd(['reset', '--hard', 'origin'], repository_path)
+        run_git_cmd(['pull'], repository_path)
+
+    return repository_path
+
+
+def get_application_git_url(app_name):
+    return 'git@github.com:alphagov/digitalmarketplace-{}.git'.format(app_name)
```

Comment:
Can we use the repository URL as the command argument instead? That way we only rely on the `digitalmarketplace-{app_name}` convention and can deploy from other clones if needed